components/Navigation/SectionNavigation.tsx

```diff
@@ -37,7 +37,13 @@ const useStyles = createUseStyles({
 type SectionType = keyof typeof sections;
 type SectionItem = { title: string; href: string };
 
-const SectionNavigation = () => {
+const SectionNavigation = ({
+  width,
+  height,
+}: {
+  width?: string | number;
+  height?: string | number;
```

Comment:
I don't see these props used in this component.

Reply:
<CommandLineIcon width="16" height="16" />